modules/psap-special-resource-operator-using-CRD.adoc

```diff
@@ -0,0 +1,404 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+:_content-type: PROCEDURE
+[id="deploy-simple-kmod-using-crd"]
+= Building and running the simple-kmod SpecialResource for a hub and spoke topology
```

Comment:
[Throughout] "hub-and-spoke" is a compound adjective
```suggestion
= Building and running the simple-kmod SpecialResource for a hub-and-spoke topology
```

Reply:
changed all instances